src/app/utils/status-message.ts

```diff
@@ -37,3 +37,52 @@ export function getMatchesAndParts(message: string) {
   const parts: string[] = message.split(numberPattern);
   return { matches, parts };
 }
+
+const responseStatusTextMap = new Map([
```

Comment:
We don't need to specify the type in the variable name because the editor gives us that information. Consider renaming `responseStatusTextMap` to `repsonseStatusText`. On the same note, `httpStatusDescription` is a more appropriate variable name.

Reply:
On it